plugins/module_utils/foreman_helper.py

```diff
@@ -617,6 +617,15 @@ def fail_from_exception(self, exc, msg):
                 fail['error'] = exc.response.text
         self.fail_json(**fail)
 
+    def handle_organization_param(self, entity_dict):
+        entity_dict = entity_dict.copy()
+
+        entity_dict['organization'] = self.find_resource_by_name('organizations', name=entity_dict['organization'], thin=True)
```

Comment:
no `if absent` because katello actually needs it always, and hostgroup has it in an if already.

nice when I am so consistent with myself and my reviews.

Reply:
Maybe leave a comment for future you.